src/site/static/eligibility.liquid

```diff
@@ -1871,8 +1871,29 @@ pageClass: "page-public-assistance"
   }
 
   function gaEligible() {
-    // TODO
-    return false;
+    // https://socialservices.sccgov.org/financial-assistance/financial-assistance-adults-without-dependent-children-general-assistance
+    const age = document.getElementById("age").value || null;
+    const numDependents = document.querySelectorAll('input[id^="hh-member-dependent"]:checked').length;
+  
+    const MIN_GA_ELIGIBLE_AGE = 18;  // Years
+    const MAX_GA_ELIGIBLE_AGE = 64;  // Years
```

Comment:
Section 5.1 [here](https://stgenssa.sccgov.org/debs/policy_handbook_GA/gachap05.pdf) suggests people can be older than 64...I think?  The language is confusing, and I'm curious what your interpretation is.